src/Microsoft.AspNet.Hosting/Program.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNet.Hosting
 {
     public class Program
     {
-        private const string HostingIniFile = "Microsoft.AspNet.Hosting.ini";
+        private const string HostingJsonFile = "Microsoft.AspNet.Hosting.json";
```

Comment:
Perhaps it would be better to simply support both?  Prefer json first, if not present, look for ini?